### DISCUSSION TITLE
Problem opening WRL/VRML file

### DISCUSSION THREAD

**Discussion by adamalfath** [fougue/mayo#267] [Q&A] [answered]
Hi, I'm new mayo user. I have a problem opening WRL/VRML files (file transfer problem). The file is generated using a [script](https://github.com/easyw/kicadStepUpMod) from a STEP file. STEP file is opened just fine on mayo. Exporting it to VRML from mayo and opening it again will have the same error.

I'm using v0.8.0 665e386

[LCM_ZJY096T-IG11.zip](https://github.com/fougue/mayo/files/14881631/LCM_ZJY096T-IG11.zip)

**Comment by HuguesDelorme** [comment]
Hello @adamalfath
The problem you reported has been reproduced, this is caused by confusion of VRML to be read as STL file
This will be fixed soon

**Reply by HuguesDelorme** [reply, marked as answer]
The issue reported has been fixed in `develop` branch(see #268 for more info)

**Reply by adamalfath** [reply]
Thank you for your support and for catching the additional problem. Strange it never thrown an error by other software I use so far (freecad, f3d, view3dscene, qiew)
